core/src/main/resources/error/error-classes.json

```diff
@@ -235,6 +235,23 @@
     ],
     "sqlState" : "22018"
   },
+  "INSUFFICIENT_TABLE_PROPERTY": {
+    "message" : [
+      "Can't find table property:"
+    ],
+    "subClass" : {
+      "TABLE_PROPERTY_MISSING" : {
```

Comment:
No need to duplicate `TABLE_PROPERTY_` in sub-classes.

Reply:
Yep, fixed